docker/official/remco/templates/jaas-loginmodule.conf

```
@@ -22,6 +22,7 @@
         rolePrefix="{{ getv("/rundeck/jaas/ldap/rolePrefix", "") }}"
         cacheDurationMillis="{{ getv("/rundeck/jaas/ldap/cachedurationmillis", "600000") }}"
         reportStatistics="{{ getv("/rundeck/jaas/ldap/reportstatistics", "true") }}"
+	roleUsernameMemberAttribute="{{ getv("/rundeck/jaas/ldap/rolessernamememberattribute", "memberUid") }}"
```

We may need to wrap this in an `if exists` as the documentation states it overrides `roleMemberAttribute`
```suggestion
    {% if exists("/rundeck/jaas/ldap/roleusernamememberattribute") -%}
        roleUsernameMemberAttribute="{{ getv("/rundeck/jaas/ldap/roleusernamememberattribute") }}"
    {% endif %}
```